src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -302,4 +321,86 @@ public void test() throws IOException {
             executableSection.execute(restTestExecutionContext);
         }
     }
+
+    // don't look any further: NO TOUCHY!
+    
+    public static class Rest0Tests extends ElasticsearchRestTestCase {
```

Comment:
@rmuir that commented code is irrelevant, it was never turned on nor tested cause it would mess up reporting, the way you do it now is the only way we have for now I believe. I hoped we could fix the out-of-the-box distributions at some point but it required some deeper changes in junit reporting.